src/pages/LendingTeam/TeamListing.vue

```diff
@@ -0,0 +1,312 @@
+<template>
+	<div>
+		<div class="tw-flex
+		tw-justify-between
+		tw-items-baseline
+		tw-mb-4"
+		>
+			<h2 class="tw-text-h3">
+				Team Listing
+			</h2>
+			<p class="tw-text-small">
+				{{ numeral(totalCount).format('0,0') }} lending teams in all categories
+			</p>
+		</div>
+		<team-search-bar @search="handleSearchQuery"
+			id="team-list-search-bar"
+		/>
+		<div class="md:tw-flex
+		md:tw-flex-row
+		tw-items-baseline
+		tw-mb-4
+		md:tw-justify-between"
+		>
```

Comment:
These  filter dropwdowns respond to the screen size changes but do not shrink entire element on tablet and stack on mobile (which is fine, they are just not spanning the screen)